src/queries/self/registration/useGetEventAttendeeTransfersLogs.ts

```diff
@@ -11,31 +10,29 @@ import {
 
 export const EVENT_PASS_TRANSFER_LOGS_QUERY_KEY = (
   eventId: string,
-  passId: string,
-  transferLogId?: string
-): QueryKey => ["EVENT_PASS_TRANSFER_LOGS", eventId, passId, transferLogId];
+  attendeeId: string
+): QueryKey => ["EVENT_PASS_TRANSFER_LOGS", eventId, attendeeId];
```

Comment:
The key should be `[...SELF_EVENT_REGISTRATION_QUERY_KEY(eventId, attendeeId), "TRANSFERS"]`